Dockerfile

```diff
@@ -82,6 +87,14 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install Criu
+ENV CRIU_VERSION 1.7
```

Comment:
@boucher may be able to give more input, but I think there were problems in CI, and going back to 1.7 solved them (https://github.com/docker/docker/pull/13602#issuecomment-164890260). I assume it can be a longer term item to resolve what version is best.